src/services/EdFi.DataManagementService.Api.Tests.E2E/Management/ContainerSetup.cs

```diff
@@ -11,21 +11,34 @@ public class ContainerSetup
 {
     public async Task<string> SetupDataManagement()
     {
-        string imageName = "local/edfi-data-management-service";
+        var network = new NetworkBuilder().Build();
+
+        string apiImageName = "local/edfi-data-management-service";
+        string dbImageName = "local/edfi-data-management-postgresql";
 
         // Image needs to be previously built
-        var dockerImage = new ContainerBuilder()
-            .WithImage(imageName)
+        var apiContaner = new ContainerBuilder()
+            .WithImage(apiImageName)
             .WithPortBinding(8080)
             .WithWaitStrategy(Wait.ForUnixContainer().UntilHttpRequestIsSucceeded(r => r.ForPort(8080)))
+            .WithNetwork(network)
             .Build();
 
-        await dockerImage.StartAsync();
+        var dbContainer = new ContainerBuilder()
+        .WithImage(dbImageName)
+        .WithPortBinding(5432, 5432)
```

Comment:
In another ticket - let's get this one done first - I would like to use a different port number, so that we can run E2E tests locally without conflicting with another instance of PostgreSQL that is already running.

Reply:
Come to think of it, this might not be a problem. I don't know if TestContainers is exposing this port to localhost; perhaps it is only internal on the Docker network.

Reply:
@stephenfuqua I haven't tried it since I added that network wrapper around our test containers. Before then though - yeah I was starting and stopping a lot.